src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -69,6 +69,9 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
+    // Extension for collection argument list files
+    private static final String COLLECTION_LIST_FILE_EXTENSION = ".list";
```

Comment:
Maybe we need a `--helphelp` option.  In all seriousness though, being able to control help verbosity would be nice... We could do the` -h`, `-hh`, `-hhhhhhhh` thing that some people use for verbosity, although that's a bit funky.